tests/living_documentation_test.py

```diff
@@ -1,12 +1,13 @@
 import boto3
-from retrying import retry
+import pytest
+import botocore
 
 cw = boto3.client('cloudwatch')
 lambda_client = boto3.client('lambda')
 
 
 def test_superwerker_dashboard():
```

Comment:
we are not testing if the contents are properly displayed?

Reply:
Good point. I added another test.